bigbluebutton-html5/imports/api/voice-users/server/handlers/joinVoiceUser.js

```diff
@@ -1,12 +1,82 @@
 import { check } from 'meteor/check';
 
+import Logger from '/imports/startup/server/logger';
+import Users from '/imports/api/users';
 import addVoiceUser from '../modifiers/addVoiceUser';
 
 export default function handleJoinVoiceUser({ body }, meetingId) {
   const voiceUser = body;
   voiceUser.joined = true;
 
   check(meetingId, String);
+  check(voiceUser, {
+    voiceConf: String,
+    intId: String,
+    voiceUserId: String,
+    callerName: String,
+    callerNum: String,
+    muted: Boolean,
+    talking: Boolean,
+    callingWith: String,
+    listenOnly: Boolean,
+    joined: Boolean,
+  });
+
+  const {
+    intId,
+    callerName,
+  } = voiceUser;
+
+  if (intId.toString().startsWith('v_')) {
+    /* voice-only user - called into the conference */
+
+    const selector = {
+      meetingId,
+      userId: intId,
+    };
+
+    const USER_CONFIG = Meteor.settings.public.user;
+    const ROLE_VIEWER = USER_CONFIG.role_viewer;
+
+    const modifier = {
+      $set: {
+        meetingId,
+        connectionStatus: 'online',
+        roles: [ROLE_VIEWER.toLowerCase()],
+        sortName: callerName.trim().toLowerCase(),
+        color: '#ffffff', // TODO
```

Comment:
will be addressed in Make call-in/dial-in users visually different than web users #4786